#206 view date picker for task starting date

Ekaji opened the pull request:
date picker for selecting the start date of task. 
1. click on the date input to open a calendar 
2. selecting a date auto closet the calendar
3. on select chosen date is visible

Ekaji committed:
#206 view datepicker for task starting date

a11rew committed:
Merge branch 'dev' of https://github.com/zurichat/zc_plugin_reminder into date-picker-for-start-date

a11rew reviewed (changes requested):
Looks good, couple of updates and you're good;
- Modify the design to change on smaller screens using Tailwind breakpoints. Refer to the figma design for what the date picker should look like on mobile.
- Questionable useEffect but we'll let it slide for now.
- Download the svg provided by the designer in the figma mockup, put it in the assets folder, import it and use that instead of the remote url.
- You can't set the width of the container with tailwind because you already set it in the same className variable with `w-full`. The only reason the inline style works instead is because it overrides the `w-full` style you set. If you want the width of container to fill parent use w-full (Recommended) else use w-72 or w-80 if you need around 300px

a11rew committed:
update style and onChange function

Ekaji committed:
#206 removed hard coded style, used design provided calendar.svg cursor pointer and date input invisible on small screen

removed hard coded style, used design provided calendar.svg cursor pointer and date input invisible on small screen

Ekaji commented:
i have made the sudgested changes i also added click event to the calender icon and added cursor on hover

Ekaji committed:
#206 removed hard coded style, used design provided calendar.svg cursor pointer and date input invisible on small screen

removed hard coded style, used design provided calendar.svg cursor pointer and date input invisible on small screen

a11rew committed:
Merge branch 'dev' of https://github.com/zurichat/zc_plugin_reminder into date-picker-for-start-date

a11rew committed:
renamed Component

a11rew reviewed (approved):
Nice work 👍
